### PR TITLE
ISPN-14214 Redis connector simple tutorial

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-14214

### CODE REVIEW

File: infinispan-remote/redis-client/src/main/java/org/infinispan/tutorial/simple/redis/RedisClientCache.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package org.infinispan.tutorial.simple.redis;
+
+import io.lettuce.core.RedisClient;
+import io.lettuce.core.api.sync.RedisCommands;
+import org.infinispan.client.hotrod.impl.ConfigurationProperties;
```
Comment (tristantarrant) [inline infinispan-remote/redis-client/src/main/java/org/infinispan/tutorial/simple/redis/RedisClientCache.java:5]: I would avoid referencing a Hot Rod class (even if it's just a constant)

Comment (wburns) [reply infinispan-remote/redis-client/src/main/java/org/infinispan/tutorial/simple/redis/RedisClientCache.java:5]: Maybe add a "SINGLE_PORT" class variable to `Infinispan` instead or something?

**Review by wburns — APPROVED** [review]
Seems good to me.

**Comment by karesti** [comment]
@tristantarrant I removed infinispan hotrod link but left the constants to avoid any duplication

File: infinispan-remote/redis-client/pom.xml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    <dependencies>
+        <dependency>
+            <groupId>org.infinispan.tutorial.simple</groupId>
+            <artifactId>connect-to-infinispan-server</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.lettuce</groupId>
+            <artifactId>lettuce-core</artifactId>
+            <version>6.2.1.RELEASE</version>
```
Comment (pruivo) [inline infinispan-remote/redis-client/pom.xml:48]: we are using 6.2.0
```suggestion
            <version>6.2.0.RELEASE</version>
```

**Review by pruivo — CHANGES_REQUESTED** [review]

File: infinispan-remote/redis-client/pom.xml (hunk 2 above)
Comment (karesti) [reply infinispan-remote/redis-client/pom.xml:48]: This is the client side, we can use the latest version or any other redis client

**Comment by karesti** [comment]
@pruivo I changed the comment to RESP

**Comment by karesti** [comment]
@pruivo feel free to merge please :)))

**Comment by wburns** [comment]
Integrated into main, thanks @karesti !

**Pull request merged**